streamduck-daemon/src/main.rs

```diff
@@ -171,6 +172,10 @@ async fn root(matches: ArgMatches) {
         std::process::exit(0);
     });
 
+    if config.autosave == true {
```

Comment:
Boolean comparison? The config.autosave is already a boolean
```suggestion
    if config.autosave() {
```
(Suggesting according to comment about autosave)